scenarios/kubernetes_bazel.py

```diff
@@ -190,6 +190,8 @@ def main(args):
         if test_pkgs or manual_test_targets or affected:
             tests = bazel.query('test', test_pkgs, affected) + manual_test_targets
             if tests:
+                if args.test_config:
+                    tests = ['--config='+args.test_config] + tests
```

Comment:
is there a strong advantage to doing this rather than just adding `--config=unit` to the test_args option?

Reply:
It seemed maybe more obvious to have a --config flag but in retrospect this is specific to the test portion anyhow so I will do this.

Reply:
updated